### PR TITLE
Wishlist v2

### PR DESCRIPTION
**Pull request by dipen2000**
Please review the wishlist feature of my E-comm app.

### CODE REVIEW

File: src/Components/Cards/CategoryCard/CategoryCard.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import "./CategoryCard.css";
+const CategoryCard = ({ category, onClick }) => {
+  return (
+    <div className="category-card-container curs-point" onClick={onClick}>
+      <img
+        className="img-resp"
+        src={category.categoryImg}
+        alt={`${category.categoryName} collection img`}
```
Comment (Shivani-1524) [inline src/Components/Cards/CategoryCard/CategoryCard.js:8]: suggestion: remoce 'img' from alt, as it automatically screen reads with image

File: src/Components/Chip/Chip.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.chip-container{
+    border : 3px solid black;
+    width : 8rem;
+    height : 5rem;
+}
+
+.chip-container:hover{
+    background-color: rgba(0, 0, 0, 0.112);
```
Comment (Shivani-1524) [inline src/Components/Chip/Chip.css:8]: suggestion: use color variables

File: src/Components/InputFields/PasswordField/PasswordField.css [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    border : 3px solid black;
+    width : 30rem;
+}
+
+.input-field{
+    padding : 0.5rem;
+}
+
+.input-field-icon{
+    /* border : 3px solid red;  */
```
Comment (Shivani-1524) [inline src/Components/InputFields/PasswordField/PasswordField.css:11]: remove comments

File: src/Components/Navbar/Navbar.js [hunk 4]
```diff
@@ -1,31 +1,51 @@
...
-}
+const Navbar = () => {
+  const navigate = useNavigate();
+  const { isAuth } = useAuth();
+  const { wishlistState } = useWishlist();
+  return (
+    <header className="navbar-sticky">
+      <nav className="navbar flex-row justify-space-between-flex align-center-flex">
+        <Link to="/" className="curs-point">
+          <h2>SHOEDOG</h2>
```
Comment (Shivani-1524) [inline src/Components/Navbar/Navbar.js:15]: its not recommended to use all caps letters for accessiblity reasons.

File: src/Components/Navbar/Navbar.js [hunk 5]
```diff
@@ -1,31 +1,51 @@
 import "./Navbar.css";
-import { Link } from "react-router-dom";
+import { Link, useNavigate } from "react-router-dom";
+import { ButtonPrimary } from "../Buttons";
+import { useAuth } from "../../context/authContext";
+import { useWishlist } from "../../context/wishlistContext";
```
Comment (rahulyadav139) [inline src/Components/Navbar/Navbar.js:5]: Suggestion: use barrel exporting.

File: src/Pages/Home/Home.js [hunk 6]
```diff
@@ -1,10 +1,63 @@
...
+            className="img-resp"
+            src="https://i.postimg.cc/DwNskF02/daniel-storek-JM-q-KEd1-GMI-unsplash.jpg"
+            alt="Hero img"
+          />
+          <div className="flex-col hero-CTA-section">
+            <h2 className="heading-2">Get you shoe game up!</h2>
+            <div>
+              <ButtonPrimary onClick={() => navigate("/product")}>
+                Shop now
+              </ButtonPrimary>
```
Comment (rahulyadav139) [inline src/Pages/Home/Home.js:28]: Suggestion: use Link component if you want to navigate only.

**Review by rahulyadav139 — APPROVED** [review]

**Pull request merged**